lib/matplotlib/ticker.py

```diff
@@ -931,51 +932,35 @@ def label_minor(self, labelOnlyBase):
         """
         self.labelOnlyBase = labelOnlyBase
 
+    def _numdec(self, vmin, vmax):
```

Comment:
Moved here so `SymmetricalLogLocator` can override the decade determination code.

Reply:
the method could use a nicer name (say, `_number_of_decades`) and then you don't need the docstring :)